test/bntest.c

```diff
@@ -3165,6 +3165,33 @@ static int test_mod_exp2_mont(void)
     return res;
 }
 
+static int test_mod_inverse(void)
+{
+    int res = 0;
+    char *str = NULL;
+    BIGNUM* a = NULL;
+    BIGNUM* b = NULL;
+
+    if (!TEST_true(BN_dec2bn(&a, "5193817943")))
+        goto err;
+    if (!TEST_true(BN_dec2bn(&b, "3259122431")))
+        goto err;
+    if (!TEST_ptr_eq(BN_mod_inverse(b, a, b, ctx), b))
```

Comment:
A comment stating that we are testing aliased parameters might be useful here

Reply:
done.